pkg/operator/controllers/const.go

```diff
@@ -5,7 +5,7 @@ package controllers
 
 const (
 	AlertwebhookControllerName             = "Alertwebhook"
-	AzureNSGControllerName                 = "AzureNSG"
+	AzureSubnetsControllerName             = "AzureNSG"
```

Comment:
Does the value of this constant need to remain `AzureNSG`?

Reply:
no :)